src/lib/checkbox/checkbox.spec.ts

```diff
@@ -542,6 +543,88 @@ describe('MatCheckbox', () => {
         expect(checkboxNativeElement).not.toMatch(/^mat\-checkbox\-anim/g);
       });
     });
+
+    describe('click action checked only setting', () => {
+      beforeEach(() => {
+        TestBed.resetTestingModule();
+        TestBed.configureTestingModule({
+          imports: [MatCheckboxModule, FormsModule, ReactiveFormsModule],
+          declarations: [
+            SingleCheckbox,
+          ],
+          providers: [
+            {provide: MAT_CHECKBOX_CLICK_ACTION, useValue: 'check'},
+            {provide: ViewportRuler, useClass: FakeViewportRuler}
+          ]
+        });
+
+        fixture = TestBed.createComponent(SingleCheckbox);
+        fixture.detectChanges();
+
+        checkboxDebugElement = fixture.debugElement.query(By.directive(MatCheckbox));
+        checkboxNativeElement = checkboxDebugElement.nativeElement;
+        checkboxInstance = checkboxDebugElement.componentInstance;
+        testComponent = fixture.debugElement.componentInstance;
+
+        inputElement = <HTMLInputElement>checkboxNativeElement.querySelector('input');
+        labelElement = <HTMLLabelElement>checkboxNativeElement.querySelector('label');
```

Comment:
Prefer `as HTMLElement` (even if the other tests use the other syntax from before we had that rule)